test/unit/spec/EnrollEmail_spec.js

```diff
@@ -0,0 +1,180 @@
+/* eslint max-params: 0 */
+define([
+  'q',
+  'okta',
+  '@okta/okta-auth-js/jquery',
+  'util/Util',
+  'helpers/mocks/Util',
+  'helpers/dom/AuthContainer',
+  'helpers/dom/EnrollEmailForm',
+  'helpers/dom/EnrollActivateEmailForm',
+  'helpers/dom/Beacon',
+  'helpers/util/Expect',
+  'sandbox',
+  'LoginRouter',
+  'helpers/xhr/MFA_ENROLL_email',
+  'helpers/xhr/MFA_ENROLL_ACTIVATE_email',
+  'helpers/xhr/SUCCESS',
+], function (Q, Okta, OktaAuth, LoginUtil, Util, AuthContainer,
+  EnrollEmailForm, EnrollActivateEmailForm, Beacon, Expect, $sandbox, Router,
+  xhrEnrollEmail, xhrEnrollActivateEmail, xhrSUCCESS) {
+
+  var { $ } = Okta;
+  var itp = Expect.itp;
+
+  Expect.describe('EnrollEmail', function () {
+
+    function setup (resp) {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'http://localhost:3000';
+      var authClient = new OktaAuth({
+        url: baseUrl,
+        transformErrorXHR: LoginUtil.transformErrorXHR
+      });
+      var successSpy = jasmine.createSpy('successSpy');
+      var router = new Router({
+        el: $sandbox,
+        baseUrl: baseUrl,
+        authClient: authClient,
+        globalSuccessFn: successSpy,
+        'features.router': true
+      });
+      // router.on('afterError', afterErrorHandler);
+      Util.registerRouter(router);
+      Util.mockRouterNavigate(router, true);
+      setNextResponse(resp);
+      return Util.mockIntrospectResponse(router)
+        .then(function () {
+          router.refreshAuthState('dummy-token');
+          return Expect.waitForEnrollChoices();
+        })
+        .then(function () {
+          router.enrollEmail();
+          return Expect.waitForEnrollEmail({
+            router: router,
+            authContainer: new AuthContainer($sandbox),
+            beacon: new Beacon($sandbox),
+            form: new EnrollEmailForm($sandbox),
+            setNextResponse: setNextResponse,
+            successSpy: successSpy,
+          });
+        });
+    }
+
+    itp('display start enroll email form and is able to send code to email', function () {
+      return setup(xhrEnrollEmail)
+        .then(function (test) {
+          expect(test.form.titleText()).toBe('Set up Email Authentication');
+          expect(test.form.enrollEmailContent()).toBe('Send a verification code to your registered email.');
+          expect(test.form.submitButtonText()).toBe('Send me the code');
+          return test;
```

Comment:
do we really need this `return`? Can we get away deleting lines 70-72? Asking since there are no `waitFor`'s here

Reply:
we could. I was trying to separate assertions: assert form elements v.s. ajax call.

Reply:
either works. I was just curious if there was some technical reason.

Reply:
no technical reason at all.